index.html

```diff
@@ -243,12 +256,17 @@ <h3 class="connectlink">LET´S CONNECT</h3>
             </ul>
           </div>
           <div class="link-project"></div>
-          <a class="linktonic" href="#">Get my resume</a>
+<div class="resume-link">
+          <a href="#">Get my resume</a>
```

Comment:
- [ ] Kindly fix the resume button when is pressed, : hover

Reply:
- [ ] Kindly fix hover in button. This is code color for hover button: `#6070FF;`
- [ ] [OPTIONAL] Consider fixing the hover color for other buttons also.